feature/bgp/feature.textproto

```diff
@@ -17,6 +17,20 @@ id {
   version: 1
 }
 
+# Network Instance config
```

Comment:
These paths already existing in NI base. We should just create a dependency on NI base rather than duplicating the paths.